src/components/Filter/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+
+function Form() {
+    const [values, setValues] = React.useState({}); 
+
+    const onChange = (event: React.SyntheticEvent) => {
+        event.persist();
+        const target = event.target as HTMLInputElement;
+
+        setValues(values => ({...values, [target.name]: target.value}));
+    }
+    const submit = async (event: React.SyntheticEvent) => {
+        event.preventDefault()
+    
+        const res = await fetch('/api/filter', {
```

Comment:
Are we wanting to keep track of api routes in `utils/urls.ts?`

Reply:
Yes. Some things in next require using absolute routes, so having URL file makes it easy to implement those. It also makes the codebase easier to maintain in the event that one of the routes changes at any point.